truffle/com.oracle.truffle.api/src/com/oracle/truffle/api/impl/Accessor.java

```diff
@@ -163,6 +163,16 @@ protected Object eval(TruffleLanguage<?> l, Source s, Map<Source, CallTarget> ca
         return API.eval(l, s, cache);
     }
 
+    /**
+     * @param vm
+     * @param ev
+     * @param code text to evaluate
+     * @param node node in halted context
+     * @param frame frame in halted context
+     * @return result of language evaluation
+     * @throws IOException
+     * @throws UnsupportedOperationException if not supported by the language implementation
+     */
     protected Object evalInContext(Object vm, Object ev, String code, Node node, MaterializedFrame frame) throws IOException {
```

Comment:
No, but it seems redundant (if you change the API method's javadoc, you'd have to change it in two places), and the `@param vm` and `@param ev` parts are useless without description. Alternatively, you could put a `@see APIClass#documentedMethod` there instead.